core/controllers/suggestion.py

```diff
@@ -21,13 +21,35 @@
 from constants import constants
 from core.controllers import acl_decorators
 from core.controllers import base
+from core.domain import opportunity_services
 from core.domain import suggestion_services
 from core.platform import models
 import feconf
+import utils
 
 (suggestion_models,) = models.Registry.import_models([models.NAMES.suggestion])
 
 
+def _check_valid_suggestion_type_and_target_type(target_type, suggestion_type):
```

Comment:
_require_suggestion_and_target_types_to_be_valid or _require_suggestion_type_and_target_type_to_be_valid or _require_valid_suggestion_and_target_types (the last one probably reads best)

(We typically start with "require" when the failure of a check causes an exception to be thrown)

Reply:
Done!